ui/src/main/java/org/pentaho/di/ui/core/events/dialog/SelectionAdapterFileDialog.java

```diff
@@ -40,6 +40,7 @@
 import org.pentaho.di.ui.core.FileDialogOperation;
 import org.pentaho.di.ui.core.events.dialog.extension.ExtensionPointWrapper;
 import org.pentaho.di.ui.core.events.dialog.extension.SpoonOpenExtensionPointWrapper;
+import org.pentaho.di.ui.spoon.Spoon;
```

Comment:
Please update the copyright date on this file.  Looks ok otherwise.

Reply:
Addressed review comments